### PR TITLE
Add update_to_l2 command

### PR DESCRIPTION
**Pull request by Uxio0**
Updates a v1.1.1/v1.3.0/v1.4.1 non L2 Safe to a L2 Safe supported by Safe Wallet UI.
The migration contract address needs to be provided.
It can be found [here](https://github.com/safe-global/safe-contracts/blob/main/contracts/libraries/SafeToL2Migration.sol).
Nonce for the Safe must be 0 and supported versions are v1.1.1, v1.3.0 and v1.4.1.

### CODE REVIEW

File: safe_cli/operators/utils/safe_to_l2_migration.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+safe_to_l2_migration = {
+    "_format": "hh-sol-artifact-1",
+    "contractName": "SafeToL2Migration",
```
Comment (moisses89) [inline safe_cli/operators/utils/safe_to_l2_migration.py:3]: Maybe can be better move this file to a contracts folder, utils is very generic.

File: safe_cli/operators/safe_operator.py [hunk 2]
```diff
@@ -586,6 +588,54 @@ def update_version(self) -> Optional[bool]:
...
+                f"Non L2 to L2 migration contract {migration_contract_address} is not deployed"
+            )
+
+        safe_version = self.safe.retrieve_version()
+        chain_id = self.ethereum_client.get_chain_id()
+
+        if self.safe.retrieve_nonce() > 0:
+            raise InvalidNonceException("Nonce must be 0 for non L2 to L2 migration")
+
+        if safe_version == "1.1.1":
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply safe_cli/operators/safe_operator.py:613]: Yes, there's no 1.1.1 L2

File: safe_cli/operators/utils/safe_to_l2_migration.py (hunk 1 above)
Comment (Uxio0) [reply safe_cli/operators/utils/safe_to_l2_migration.py:3]: Yes, I'm thinking that this might belong to safe-eth-py

**Review by moisses89 — APPROVED** [review]

**Comment by Uxio0** [comment]
@JagoFigueroa we require QA on this 🙏🏻 . The way to test it would be to deploy a Safe v1.1.1 and v1.3.0 (not L2) on Polygon, use the command `update_to_l2 0x1Ba30910CE8Eb45b276aa7c89C38FC52228B85CB` (that's the address for the not official migration contract in Polygon deployed by me) and then use the Safe with our official UI and check transactions show as expected

**Pull request merged**